README.md

```diff
@@ -220,21 +220,18 @@ An IdP sending the `Login-Status: logged-in` response header refreshes the expir
 
 Sending `Login-Status: logged-out` clears the profile information along with the login status bit.
 
-### Distinguishing between `sign-up` and `sign-in` without an `accounts_endpoint`
+### `approved_clients`: Partially Supported
 
 By default, FedCM makes a distinction between `sign-up` and `sign-in` via a property in the user profile information, called `approved_clients`, that is received in the [fetch the accounts](https://w3c-fedid.github.io/FedCM/#fetch-the-accounts) step: if the `clientId` passed in the `navigator.credentials.get()` call is not a member of `approved_clients`, it means that this client was never previously approved by the user.
 
-This could also be accomplished client-side by passing a list of `approved_clients` in the login status API call:
+This functionality should not be impacted by either the "Optional Endpoints" or "Config Push" features, but does have an obvious correct behavior for the "Accounts Push" feature.
 
-```js
-navigator.login.setStatus("logged-in", {
-	accounts: [{
-         // ... other fields
-         approved_clients: ["https://rp.example"],
-         // ... other fields
-  }]
-});
-```
+The `approved_clients` property is only useful if it is reasonably fresh. In order for `approved_clients` to be useful with Accounts Push, it would require either:
```

Comment:
```suggestion
The `approved_clients` property is only useful if it is reasonably fresh. For `approved_clients` to be useful with Accounts Push, it would also either of the following:
```

Reply:
Took this suggestion with modifications.